whisper/utils.py

```diff
@@ -44,9 +45,9 @@ def format_timestamp(seconds: float):
 
 
 def write_vtt(transcript: Iterator[dict], file: TextIO):
-    print("WEBVTT\n", file=file)
+    logging.info("WEBVTT\n", file=file)
     for segment in transcript:
-        print(
+        logging.info(
```

Comment:
Did you test this code? As far as I'm aware the logging library does not support writing to a file stream in the same way print does. So this code will likely not work.